Mathlib/Order/RelSeries.lean

```diff
@@ -477,6 +477,14 @@ def eraseLast (p : RelSeries r) : RelSeries r where
 
 @[simp] lemma last_eraseLast (p : RelSeries r) :
     p.eraseLast.last = p ⟨p.length.pred, Nat.lt_succ_iff.2 (Nat.pred_le _)⟩ := rfl
+
+/-- In a non-trivial series `p`, the last element of `p.eraseLast` is related to `p.Last` -/
+lemma eraseLast_last_rel_last (p : RelSeries r) (h : 0 < p.length) :
```

Comment:
I think we adopted the policy that hypotheses should be of the form x \ne 0, not 0 < x

Reply:
Thanks! Will update

Reply:
Thanks to `omega`, no changes needed in the proof nor its use in my project. Nice.
```suggestion
lemma eraseLast_last_rel_last (p : RelSeries r) (h : p.length ≠ 0) :
```